forms-shared/tests/sharepoint/__snapshots__/getValuesForSharepoint.ts.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`getValuesForFields should match snapshots 1`] = `
+{
```

Comment:
Isn't it missing all the fields except "non-Žiadateľ" ones?

Reply:
Yes, but I can somehow add there a snapshot test for children and other records from the form.